psh.proto

```diff
@@ -1,12 +1,25 @@
 syntax ="proto3";
 
-import "google/protobuf/empty.proto";
 package psh;
 
+import "google/protobuf/empty.proto";
+
 service PshService {
-    rpc Heartbeat(google.protobuf.Empty) returns (PshResponse);
+  rpc Heartbeat(google.protobuf.Empty) returns (PshResponse);
+  rpc SendHostInfo(HostInfoRequest) returns (HostInfoResponse);
 }
 
 message PshResponse {
-    string resp = 1;
-}
\ No newline at end of file
+  string resp = 1;
+}
+
+message HostInfoRequest {
+  optional string ipv4 = 1;
```

Comment:
Do not use `string` type for all fields inisde messages. For example you can specifiy ipv4 as `int32`?